boot/configure.ml

```diff
@@ -70,6 +74,10 @@ let () =
       , Arg.String (set_dir datadir)
       , "DIR where files for the share_root section are installed for the default build \
          context" )
+    ; ( "--toggles"
+      , Arg.String set_toggles
+      , "NAMES comma-separated list of configuration options to be set to 'enabled'. See \
+         the list of options in dune_config/compile_time." )
```

Comment:
When referring to source filenames in messages like this my preference is to refer to the full filename including its extension.

Reply:
Also this takes the string name of the variable and these are defined in the .ml file, but the documentation about what each variable does is in the .mli file. Something about that doesn't feel right to me. Like we have a well-documented interface in the mli file but users still have to look at the ml file in order to find out the strings to pass here.